### PR TITLE
Document avro.schema.literal property use for interpreting table data

### PR DESCRIPTION
**Pull request by Jessie212**
## Description

Document avro.schema.literal property use for interpreting table data

## Additional context and related issues

https://github.com/trinodb/trino/pull/14426

## Release notes

(x) This is not user-visible or docs only and no release notes are required.
( ) Release notes are required, please propose a release note for me.
( ) Release notes are required, with the following suggested text:

```markdown
# Section
* Fix some things. ({issue}`issuenumber`)
```

### CODE REVIEW

**Comment by mosabua** [comment]
Lets put this on hold until the PR from @m57lyra is merged and then rebase and put it into the correct location.

File: docs/src/main/sphinx/connector/hive.rst
Comment (mosabua) [inline docs/src/main/sphinx/connector/hive.rst:1066]: I think this is wrong

 ``avro.schema.literal`` table property 

I think this is a catalog configuration property

and the _ version is either a table property or a session property

**Comment by mosabua** [comment]
Maybe @colebow or @findinpath can clarify the technical aspects..

File: docs/src/main/sphinx/connector/hive.rst
Comment (findinpath) [reply docs/src/main/sphinx/connector/hive.rst:1066]: ``avro.schema.literal`` -> ``avro_schema_literal``

However this paragraph is mostly about `avro_schema_url` and how to eventually retrieve the avro schema of the table.

Comment (findinpath) [reply docs/src/main/sphinx/connector/hive.rst:1066]: @mosabua  `avro.schema.literal` and `avro.schema.url` are Hive specific table properties.

**Comment by m57lyra** [comment]
It is merged @mosabua as of today. Good to go!

**Review by findinpath — APPROVED** [review]

File: docs/src/main/sphinx/connector/hive.rst
Comment (findinpath) [inline docs/src/main/sphinx/connector/hive.rst:882]: Shouldn't we have here a reference about `avro_schema_literal` now?

File: docs/src/main/sphinx/connector/hive.rst
Comment (findinpath) [inline docs/src/main/sphinx/connector/hive.rst:873]: optional: A listing could be a bit easier to scan by the reader.

**Review by mosabua — APPROVED** [review]
Looks good.

**Comment by mosabua** [comment]
Could @ebyhr @hashhar or @findepi review and merge please

**Review by ebyhr — APPROVED** [review]

**Pull request merged**
